plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/util/FeatureDevClientUtil.kt

```diff
@@ -50,6 +51,10 @@ fun createConversation(proxyClient: FeatureDevClient): String {
         if (e is CodeWhispererRuntimeException) {
             errMssg = e.awsErrorDetails().errorMessage()
             logger.warn(e) { "Start conversation failed for request: ${e.requestId()}" }
+
+            if (e is software.amazon.awssdk.services.codewhispererruntime.model.ServiceQuotaExceededException) {
```

Comment:
is this the right version of `ServiceQuotaExceededException`?

Reply:
Yes, for this particular API `createTaskAssistConversation`, this is the correct version of  ServiceQuotaExceededException.